app/views/research_sessions/preview.html.erb

```diff
@@ -113,7 +113,7 @@
 
     <p>The data will be kept for <%= edit_link_for(:shared_duration) %>.</p>
   </section>
-  <% if @research_session.when_text.present? or @research_session.duration.present? %>
+  <% if @research_session.where_when_enabled %>
```

Comment:
I'm not sure this is a straight swap. I think we need both, or we need a validator to check that at least one value is `#present?` when `where_when_enabled` is true.

Reply:
(if we carry on like this, we're quite within our rights to say "yes" to enabling this section, then fill nothing in and have blank values in the preview)